app/controllers/posts_controller.rb

```diff
@@ -0,0 +1,8 @@
+class PostsController < ApplicationController
+  def index
+    # @users= User.find(params[:user_id])
+    # @posts = Post.where(user_id: @users)
```

Comment:
- [ ] Kindly make sure you remove the commented and unused code lines from your code base to make it more professional

Reply:
removed all commented code as suggested by the reviewer